dashboard/modules/tune/tune_head.py

```diff
@@ -0,0 +1,233 @@
+        transforms them, and serves them up over an API.
+
+        Args
+            logdir (str): Directory path to save the status information of
+                            jobs and trials.
+        """
+        super().__init__(dashboard_head)
+        self._logdir = None
+        self._trial_records = {}
+        self._trials_available = False
+        self._tensor_board_dir = ""
+        self._enable_tensor_board = False
+        self._errors = {}
+
+    @routes.get("/tune/info")
+    async def tune_info(self, req) -> aiohttp.web.Response:
+        stats = self.get_stats()
+        return await rest_response(
+            success=True, message="Fetched tune info", result=stats)
+
+    @routes.get("/tune/availability")
+    async def get_availability(self, req) -> aiohttp.web.Response:
+        availability = self._get_availability()
+        return await rest_response(
+            success=True,
+            message="Fetched tune availability",
+            result=availability)
+
+    @routes.get("/tune/set_experiment")
+    async def set_tune_experiment(self, req) -> aiohttp.web.Response:
+        try:
+            experiment = req.query["experiment"]
+        except KeyError:
+            return await rest_response(success=False, message="Bad request")
+
+        err, experiment = self.set_experiment(experiment)
+        if err:
+            return await rest_response(success=False, error=err)
+        return await rest_response(
+            success=True, message="Successfully set experiment", **experiment)
+
+    @routes.get("/tune/enable_tensorboard")
+    async def enable_tensorboard(self, req) -> aiohttp.web.Response:
+        self._enable_tensorboard()
+        if not self._tensor_board_dir:
+            return await rest_response(
+                success=False, message="Error enabling tensorboard")
+        return await rest_response(success=True, message="Enabled tensorboard")
+
+    def get_stats(self):
+        tensor_board_info = {
+            "tensorboard_current": self._logdir == self._tensor_board_dir,
+            "tensorboard_enabled": self._tensor_board_dir != ""
+        }
+        return {
+            "trial_records": copy.deepcopy(self._trial_records),
+            "errors": copy.deepcopy(self._errors),
+            "tensorboard": tensor_board_info
+        }
+
+    def set_experiment(self, experiment):
+        if os.path.isdir(os.path.expanduser(experiment)):
+            self._logdir = os.path.expanduser(experiment)
+            return None, {"experiment": self._logdir}
+        else:
+            return "Not a Valid Directory", None
+
+    def _enable_tensorboard(self):
+        if not self._tensor_board_dir:
+            tb = program.TensorBoard()
+            tb.configure(argv=[None, "--logdir", str(self._logdir)])
+            tb.launch()
+            self._tensor_board_dir = self._logdir
+
+    def _get_availability(self):
+        return {"available": True, "trials_available": self._trials_available}
```

Comment:
It seems that the `_get_availability` is only used by the `get_availability`, is it better to merge them?